plugins/auth/native/libnative.cpp

```diff
@@ -631,32 +631,32 @@ irods::auth* plugin_factory(
         AUTH_ESTABLISH_CONTEXT,
         function<error(plugin_context&)>(
             native_auth_establish_context ) );
-    nat->add_operation<rcComm_t*,const char*>(
+    nat->add_operation(
```

Comment:
Is this due to a bug in GCC?

Reply:
I couldn't tell you, as this one is impossible to google. Here's a reduced demonstration, though: https://godbolt.org/z/hr8Ysdq7E